axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -993,6 +1008,184 @@
           "address": "axelar1g03zp32gqx6qg3z59csfj4rm56lkpyqkdw9lg4qdgg6uxyupryrqcycxt9"
         }
       },
+      "XrplMultisigProver": {
+        "xrpl-2025-q1": {
+          "governanceAddress": "axelar1zlr7e5qf3sz7yf890rkh9tcnu87234k6k7ytd9",
+          "adminAddress": "axelar1lsasewgqj7698e9a25v3c9kkzweee9cvejq5cs",
+          "signingThreshold": [
+            "2",
+            "3"
+          ],
+          "serviceName": "validators",
+          "verifierSetDiffThreshold": 1,
+          "codeId": 1140,
+          "address": "axelar1etlu5c9ear5fw74c7a0kp0fkf9dqkl4dj9s3e73mf928kz8e80eqanuhua",
+          "xrplFee": 300,
+          "ticketCountThreshold": 5,
+          "availableTickets": [
```

Comment:
don't think this needs to go in the config, it can be queried on-chain if needed

Reply:
Saw the TODO to query this dynamically later

Reply:
can a range be stored instead of each ticket (assuming the full range is available)?

Reply:
Working on those TODOs to dynamically fetch those now.

Reply:
Addressed here: https://github.com/axelarnetwork/axelar-contract-deployments/pull/564/commits/7ed216eaf08d98caba48f032f1ea83990813d3d7